tests/foreman/cli/test_satellitesync.py

```diff
@@ -16,97 +16,117 @@
-from robottelo.cli.puppetmodule import PuppetModule
 from robottelo.cli.repository import Repository
 from robottelo.cli.repository_set import RepositorySet
 from robottelo.cli.settings import Settings
 from robottelo.cli.subscription import Subscription
-from robottelo.config import robottelo_tmp_dir
-from robottelo.config import settings
+from robottelo.constants import CONTAINER_REGISTRY_HUB
 from robottelo.constants import DEFAULT_CV
-from robottelo.constants import ENVIRONMENT
 from robottelo.constants import PRDS
 from robottelo.constants import REPOS
 from robottelo.constants import REPOSET
 
 
-@pytest.fixture(scope='class')
-def class_export_directory(default_sat):
-    """Create a directory for export, configure permissions and satellite settings,
-    at the end remove the export directory with all exported repository archives"""
-    export_dir = gen_string('alphanumeric')
-
-    # Create a new 'export' directory on the Satellite system
-    result = default_sat.execute(f'mkdir /mnt/{export_dir}')
-    assert result.status == 0
-
-    result = default_sat.execute(f'chown foreman.foreman /mnt/{export_dir}')
-    assert result.status == 0
-
-    result = default_sat.execute(f'ls -Z /mnt/ | grep {export_dir}')
-    assert result.status == 0
-    assert len(result.stdout) >= 1
-    assert 'unconfined_u:object_r:mnt_t:s0' in result.stdout
-
-    # Fix SELinux policy for new directory
-    result = default_sat.execute(
-        f'semanage fcontext -a -t foreman_var_run_t "/mnt/{export_dir}(/.*)?"'
-    )
-    assert result.status == 0
-
-    result = default_sat.execute(f'restorecon -Rv /mnt/{export_dir}')
-    assert result.status == 0
+EXPORT_DIR = '/var/lib/pulp/exports/'
+IMPORT_DIR = '/var/lib/pulp/imports/'
 
-    # Assert that we have the correct policy
-    result = default_sat.execute(f'ls -Z /mnt/ | grep {export_dir}')
-    assert result.status == 0
-    assert len(result.stdout) >= 1
-    assert 'unconfined_u:object_r:foreman_var_run_t:s0' in result.stdout
 
-    # Update the 'pulp_export_destination' settings to new directory
-    Settings.set({'name': 'pulp_export_destination', 'value': f'/mnt/{export_dir}'})
-    # Create an organization to reuse in tests
-    yield export_dir
-    default_sat.execute(f'rm -rf /mnt/{export_dir}')
-
-
-class ExportDirectoryNotSet(Exception):
-    """Raise when export Directory is not set or found"""
+@pytest.fixture(scope='class')
+def config_export_import_settings():
+    """Check settings and set download policy for export.  Reset to original state after import"""
+    download_policy_value = Settings.info({'name': 'default_download_policy'})['value']
+    rh_download_policy_value = Settings.info({'name': 'default_redhat_download_policy'})['value']
+    disconnected_mode_value = Settings.info({'name': 'content_disconnected'})['value']
+    assert download_policy_value == 'immediate'
+    assert rh_download_policy_value == 'on_demand'
+    assert disconnected_mode_value == 'false'
```

Comment:
what is the purpose of these assertions?

Reply:
It's a check for ensuring that those are the default sets for each of those fields.  Tests can either fail ASAP or I can add conditional statements to set it to default if it's not.  What are you thoughts?

Reply:
I personally don't think it matters what the settings were before and, since it likely doesn't matter for your tests, shouldn't be a fail condition for yours